### ISSUE TITLE
This needs to be moved inside the else block

### ISSUE THREAD

**Issue by aaronlippold** [mitre/canonical-ubuntu-16.04-lts-stig-baseline#22]
https://github.com/mitre/canonical-ubuntu-16.04-lts-stig-baseline/blob/16b03f2e0cd10cb8c2b212d8d49e38b5b81b9204/controls/V-75437.rb#L75

**Comment by kkola** [comment]
It is. The `impact` line is right after the `describe` block within the `else` block
```
  else
    describe "Not Applicable as GNOME dekstop environment is installed" do
      subject { gnome_installed }
      it { should be false }
    end
    impact 0
  end
```

**Comment by aaronlippold** [comment]
Yup, your right. Just move it above the describe block so it is a bit more
clear.

Thanks,

A
--------
Aaron Lippold

lippold@gmail.com

260-255-4779

twitter/aim/yahoo,etc.
'aaronlippold'

**Comment by aaronlippold** [comment]
Fixed by #15
